scripts/build.sh

```diff
@@ -167,3 +172,12 @@ if [ -f docker-compose.override.yml ]; then
   echo "WARNING: This may add unwanted settings to the broker that could affect how your daemon runs."
   echo ""
 fi
+
+if [ "$LOCAL" == "true" ]; then
+  echo "Downloading Local Relayer Cert..."
+  # the path of this output is directly related to the SECURE_PATH that is set
+  # in the .env file
+  curl --silent -S --output ~/.sparkswap/secure/relayer-root-regtest-local.pem http://localhost:8080/cert
```

Comment:
so no more relayer cert host? We just assume that if you're in a local set up the Relayer is on localhost? (That might be reasonable, just want to make sure I understand the choice we're making here).

Reply:
Yup. FWIW the RELAYER_CERT_HOST was never used outside of downloading the cert